source/main/gui/panels/GUI_MainSelector.h

```diff
@@ -38,12 +39,16 @@ namespace GUI {
 class MainSelector 
 {
 public:
-    const float LEFT_PANE_WIDTH = 200.f;
-    const float WINDOW_WIDTH = 600.f;
+    Ogre::Viewport* viewport = RoR::App::GetOgreSubsystem()->GetRenderWindow()->getViewport(0);
+
+    const float LEFT_PANE_WIDTH = 250.f;
+    const float WINDOW_WIDTH = viewport->getActualWidth() / 1.4;
+    const float WINDOW_HEIGHT = viewport->getActualHeight() / 1.2;
```

Comment:
TIP: read screen size from `ImGui::GetIO().DisplaySize` (example: https://github.com/RigsOfRods/rigs-of-rods/blob/4f8658fcd653bda8f0ebea05760fc9adf05a13c7/source/main/gui/panels/GUI_MultiplayerClientList.cpp#L53) - it's the same numbers, updated every frame.